adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1254,7 +1254,7 @@ String serialize(final String uniqueUserId) throws AuthenticationException {
         if (!StringExtensions.isNullOrBlank(tokenItem.getFamilyClientId())) {
             return SSOStateSerializer.serialize(tokenItem);
         } else {
-            throw new IllegalArgumentException("tokenItem does not contain family refresh token");
+            throw new UsageAuthenticationException(ADALError.FAIL_TO_EXPORT, "tokenItem does not contain family refresh token");
```

Comment:
@addev-ashish This seemed like the _most_ appropriate `ADALError`. Lmk if you have an alternative suggestion

Reply:
This looks good to me. A minor nit, can we change the condition around. !IsNull looks like a double negative to me. 

Something like 
if (isNull)
    throw UsageAuthenticationException;
// no need for else
return SSOStateSerializer.serialize(tokenItem);

Reply:
Sounds great -- will do 👍

Reply:
Fixed in 8effb3af67d98f73a808ad293bac54d65f430fce